### ISSUE TITLE
[Suggestion] Would it be possible to put an "Import" option?

### ISSUE THREAD

**Issue by mauro-cb** [steveseguin/social_stream#146]
Hello, while I was using the program, I saw the option to export a file with messages made live in TSV format.

In videos that I edit, I often put screenshots of things that were said in the chat and for some time now I have been wanting to put the viewer's avatar in these screenshots and with this TSV I saw this possibility.

I don't know of any program that makes TSV look like a chat. Would it be possible to put an "Import" option in the program that would reproduce the speeches in the saved file?

**Comment by steveseguin** [comment]
`I don't know of any program that makes TSV look like a chat. Would it be possible to put an "Import" option in the program that would reproduce the speeches in the saved file?`

Which program exactly?  Did you want social stream to import messages from a tsv file?   It might be possible, but it would be a challenge to perhaps know when the file was updated.  

There is an API for social stream already, to send messages into Social Stream, if that's of interest:
There's a page here with test buttons https://socialstream.ninja/sampleapi.html?session=YOURSTREAMIDHERE

**Comment by mauro-cb** [comment]
Yes, what I was trying to say was that it would be nice if social_stream could reproduce a TSV.

I will try to exemplify:

Let's suppose I did a live on Twitch answering the chat and I want to post an clip of it on YouTube

To do this, I enabled the option to export comments:

![Screenshot_609](https://github.com/steveseguin/social_stream/assets/62271732/a821d918-f9df-43d4-a9e1-675ec73967bf)
*It exports a TSV file

And one of the comments I received while doing live was this:

![Screenshot_607](https://github.com/steveseguin/social_stream/assets/62271732/2a97246b-01f0-44d5-bc31-2107c702f11d)

When I go to edit the video a few days later, I open the file in some TSV reader and it looks like this:

![Screenshot_608](https://github.com/steveseguin/social_stream/assets/62271732/369087f6-ea59-4071-8efd-47f19b2c9d56)

With this appearance, it is impossible to use in the video.

My intention would be to have a print with the social network icon, the person's profile photo and their nickname and message. All this in a chat aspect like the other screenshot, but this is lost when closing OBS.

It would be great to be able to send the TSV to social_stream and have it emulate the message as it was when it was sent.
